OvernightPrintsSpecTests/FFs/LogIn_Feature.feature

```diff
@@ -0,0 +1,79 @@
+﻿Feature: LogIn_Feature
+	In order to access my account on the website https://www.overnightprints.com
+	As a user of the website
+	I want to log into the website
+
+@mytag
+Scenario Template: Failure Login With Invalid Credentials

```

Comment:
Let's update the name a little bit to avoid any confusions:
Negative checks for Log In
or
Log In with invalid credentials